src/pages/ShikshalokamVoiceChat/shikshaChatStyle.css

```
@@ -380,12 +380,14 @@
 }
 
 .form-1 {
+    
     box-shadow: 0 0 15px rgba(0, 0, 0, 0.1);
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)